.code-samples.meilisearch.yaml

```
@@ -3,6 +3,39 @@
 # the documentation on build
 # You can read more on https://github.com/meilisearch/documentation/tree/master/.vuepress/code-samples
 ---
+getting_started_faceting: |-
+  var faceting = new Faceting {
+    MaxValuesPerFacet = 2
+  };
+  await client.Index("movies").UpdateFacetingAsync(faceting);
+getting_started_pagination: |-
+  var pagination = new Pagination {
+    MaxTotalHits = 500
+  };
+  await client.Index("movies").UpdatePaginationAsync(pagination);
+synonyms_guide_1: |-
+  var synonyms = new Dictionary<string, IEnumerable<string>>
+  {
+  	{ "great", new string[] { "fantastic" } },
+  	{ "fantastic", new string[] { "great" } }
+  };
+  await client.Index("movies").UpdateSynonymsAsync(synonyms);
+date_guide_index_1: |-
+  string jsonString = await File.ReadAllTextAsync("games.json");
+  var games = JsonSerializer.Deserialize<IEnumerable<Movie>>(jsonString, options);
+  var index = client.Index("games");
+  await index.AddDocumentsAsync<Movie>(games);
+date_guide_filterable_attributes_1: |-
+  List<string> attributes = new() { "release_timestamp" };
+  TaskInfo result = await client.Index("games").UpdateFilterableAttributesAsync(attributes);
```

You can simplify this one like you did below
```suggestion
date_guide_filterable_attributes_1: |-
  await client.Index("games").UpdateFilterableAttributesAsync(new string[] { "release_timestamp" });
```